app/src/main/java/edu/wisc/ece/pinpoint/utils/FirebaseDriver.java

```diff
@@ -110,6 +119,8 @@ public Task<Void> reloadAuth() {
     public Task<Void> logout(@NonNull Context context) {
         foundPinMetadata = null;
         droppedPinMetadata = null;
+        followerIds = null;
+        followingIds = null;
```

Comment:
Any reason we set this to null rather than creating a new hashset? Any reason we don't create the hashsets in the constructor? This is the case specifically for the found & dropped pin metadata since we check if they are null to know if we should fetch them or not.

Reply:
I  believe this is a remnant of debugging security rule issues

Reply:
We should make sure to reset or clear the `followerIds` and `followingIds` fields on user logout to prevent a new logged in user from having the same followers/following.